uxarray/grid/slice.py

```diff
@@ -132,4 +131,9 @@ def _slice_face_indices(grid, indices, inclusive=True):
             # drop any conn that would require re-computation
             ds = ds.drop_vars(conn_name)
 
+    if inverse_indices:
+        ds["inverse_face_indices"] = indices
+
+    ds["_is_subset"] = True
```

Comment:
Once you make the changes as described above, pass in `is_subset=True` to the constructor below.